bigtable/client/filters.h

```diff
@@ -214,26 +214,158 @@ class Filter {
    * timestamp.
    *
    * TODO(#82) - check the documentation around ordering of columns.
+   * TODO(#84) - document what is the effect of n <= 0
    */
-  static Filter CellsRowOffset(int n) {
+  static Filter CellsRowOffset(std::int32_t n) {
     Filter tmp;
     tmp.filter_.set_cells_per_row_offset_filter(n);
     return tmp;
   }
 
+  /**
+   * Return a filter that samples rows with a given probability.
+   *
+   * TODO(#84) - decide what happens if the probability is out of range.
+   *
+   * @param probability the probability that any row will be selected.  It
+   *     must be in the [0.0, 1.0] range.
```

Comment:
> [...] must be in the range [0.0, 1.0].

but you can also simplify it to:

> [...] must be in the range [0, 1].

Reply:
Did the first, I like making it explicit that is a floating point range, but will change if you feel strongly about it.

Reply:
Floating point range SGTM.